docs/components/accordion/README.md

```diff
@@ -539,6 +540,10 @@ This component has compliance with WCAG guidelines by:
 
 # API
 
+<cdr-icon class="cdr-doc-code-snippet__action-icon" use="#brand-github"/><b>View it on Github: 
```

Comment:
Use `<strong>` also missing a tag closure currently on the `<b>`

Reply:
I'm seeing this same issue on other components as well

Reply:
oh ugly - no strong or b should be there - one sec